cfgov/jinja2/v1/_layouts/layout-side-nav.html

```diff
@@ -21,8 +21,8 @@
 
 {# Adding the side nav macro #}
 {% block content_sidebar %}
-    {% import 'sidenav.html' as sidenav %}
-    {{ sidenav.render(vars.nav_items, active_nav_id) }}
+    {% from 'organisms/secondary-navigation.html' import secondary_navigation as sec_nav %}
+    {{ sec_nav(vars.nav_items, active_nav_id) }}
```

Comment:
Diverts from existing convention of `secondary_navigation.render(…`, but I guess it'll all be changing on migration to wagtail anyway.